contracts/rounds/AdjustableRoundsManager.sol

```diff
@@ -20,11 +24,11 @@ contract AdjustableRoundsManager is RoundsManager {
         num += _blocks;
     }
 
-    function blockNum() public view returns (uint256) {
+    function blockNum() public view override(RoundsManager) returns (uint256) {
```

Comment:
since this contract has a single parent, can't we just use the override keyword without mentioning the parent contract?

Reply:
RoundsManager inherits from `IRoundsManager` which is an interface, therefore `RoundsManager` has to `override` the interface definitions that are implicitly defined as `virtual`. 

Since `AdjustableRoundsManager` then inherits `RoundsManager` but `AdjustableRoundsManager` itself also overrides both the functions from `RoundsManager` and `IRoundsManager`. Therefore we need the `override` modifier here and the `virtual` modifier in the respective `RoundsManager` functions as well.